app/src/main/ui/src/pages/NewUser.tsx

```diff
@@ -0,0 +1,46 @@
+import {Button, H1} from "@cloudogu/ces-theme-tailwind";
+import React from "react";
+import {useNavigate} from "react-router-dom";
+import UserForm from "../components/users/UserForm";
+import {t} from "../helpers/i18nHelpers";
+import {useBackURL} from "../hooks/useBackURL";
+import {useSetPageTitle} from "../hooks/useSetPageTitle";
+import {emptyUser} from "../services/Account";
+import { UsersService} from "../services/Users";
+import type {User} from "../services/Users";
+
+export default function NewUser(props: { title: string }) {
+    useSetPageTitle(props.title);
+    const navigate = useNavigate();
+    const {backURL} = useBackURL();
```

Comment:
The `backURL` is never set for this page, so this can be removed. In order to make this work correctly you need to pass the `backURL` param from the users overview page when clicking the `New user` button.